domain/src/test/java/com/malibin/study/domain/lotto/ticket/LottoTicketTest.kt

```diff
@@ -0,0 +1,76 @@
+package com.malibin.study.domain.lotto.ticket
+
+import com.google.common.truth.Truth.assertThat
+import com.malibin.study.domain.lotto.LottoNumber
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.*
+
+internal class LottoTicketTest {
+
+    @ParameterizedTest
+    @MethodSource("providesInvalidLottoNumbers")
+    fun `유효하지 않은 숫자들로 LottoTicket 을 생성할 수 없다`(lottoNumbers: IntArray) {
+        // when
+        val actualException = runCatching { LottoTicket(*lottoNumbers) }.exceptionOrNull()
+
+        // then
+        assertThat(actualException).isInstanceOf(IllegalArgumentException::class.java)
+    }
+
+    @ParameterizedTest
+    @MethodSource("providesLottoNumbers")
+    fun `로또 티켓과 일치 하는 번호의 존재 여부를 확인할 수 있다`(number: Int, isSame: Boolean) {
+        // given
+        val lottoTicket = LottoTicket(1, 2, 3, 4, 5, 6)
+        val lottoNumber = LottoNumber.of(number)
+
+        // then
+        assertThat(lottoTicket.has(lottoNumber) == isSame).isTrue()
+    }
```

Comment:
비교 로직을 넣는 것 보다 직접 비교하는 것이 더 바람직해 보입니다 :)
```suggestion
        // then
        assertThat(lottoTicket.has(lottoNumber)).isEqualTo(isSame)
    }
```